AssetRipper.Library/Exporters/Textures/ImporterFactory.cs

```diff
@@ -63,6 +63,17 @@ public static ITextureImporter GenerateTextureImporter(IExportContainer containe
 			return instance;
 		}
 
+		private static int NextPowerOfTwo(int n)
+		{
+			n--;
+			n |= n >> 1;
+			n |= n >> 2;
+			n |= n >> 4;
+			n |= n >> 8;
+			n |= n >> 16;
+			return ++n;
+		}
```

Comment:
This belongs in a MathUtils class, with unit tests in the test project.

Reply:
fixed